reviews/models/mixins.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews.utils import AttrProxy
+from reviews.workflow import States
+from reviews.workflow import Workflows
+from reviews.workflow import TRANSITIONS
+
+
+class ReviewProviderSettings(models.Model):
```

Comment:
I understood, but I'm ever so mildly annoyed by the redundancy... it seems like just one more thing to remember to keep in sync. Don't have strong feelings, though, and it is a nicer interface. Meh, sure.